test/pgo_basic_SUITE.erl

```diff
@@ -204,3 +204,23 @@ types(_Config) ->
     #{command := select, rows := [Row17, Row18]} = R3,
     ?assertMatch({?TXT_UUID}, Row17),
     ?assertMatch({?TXT_UUID}, Row18).
+
+numerics(_Config) ->
```

Comment:
Could you add a test that uses a temporary table and inserts then selects a numeric.

Reply:
yeah, as soon as possible, there's not enough tests